sdk/go/pulumi-language-go/main.go

```diff
@@ -63,6 +64,8 @@ import (
 	"github.com/pulumi/pulumi/pkg/v3/codegen/schema"
 )
 
+const preferredDebugPort = 57134
```

Comment:
Consider adding a comment why we picked this number. I guess it's random, but it would be nice to see this said explicitly.